### PR TITLE
feat: Setting of Routes Skeletons (REST API)- basic CRUD (#39523)

### PR DESCRIPTION
**Pull request by nx-mukesh**
basic CRUD operation without database model.

### CODE REVIEW

**Review by namanpar — CHANGES_REQUESTED** [review]
Do review your own pr before assigning it to reviewer. You raised pr for main instead of develop
Only raise pr for develop

**Comment by nx-mukesh** [comment]
PR to develop branch

File: src/Server.ts [hunk 1]
```diff
@@ -27,15 +30,15 @@ export default class Server {
...
   public setupRoute() {
     const { app } = this;
+    const jsonParser = bodyParser.json();
 
     app.use('/health-check', (req, res) => {
       res.status(200).send('I am OK');
     });
-
+    app.use('/api', jsonParser, router);
     app.use(notFoundRoute);
```
Comment (namanpar) [inline src/Server.ts]: No need to use parser here. In bootstrap function call initBodyParser () first.

**Review by namanpar — CHANGES_REQUESTED** [review]

File: src/Server.ts (hunk 1 above)
Comment (namanpar) [reply src/Server.ts]: Do create all crud route for user as well

Comment (nx-mukesh) [reply src/Server.ts]: routes created for users and fixed initBodyParser().

Comment (namanpar) [reply src/Server.ts]: You haven't done any changes

**Review by namanpar — APPROVED** [review]

**Pull request merged**